sass/components/html/footer.html

```diff
@@ -1,11 +1,55 @@
 <footer class="footer">
-
+  <div class="footer__improved">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="footer__improved--column-first">
```

Comment:
mmhh...considering what this is, i wouldn't expect the column to contain a "menu item", this will be probably printing the site name variable and it should be an heading in my opinion, for SEO sake.
There is also this use of the modifiers here which is not correct, those are modifiers of an element that wraps them, too weird.
I don't think you need to use anything different than we have in the footer, it is still a .footer__column and i wouldn't change that, i see that you changed that even in the "main" component  but again, also there there is a wrong definition of modifiers, i would revert that.
Consider also that you divided this in 4 columns, but this is because the mockup was misleading, we agreed with Filipa that we would have had the same number of columns, the last one has to be dropped, in my opinion. 
Furthermore, i would avoid any change in the main "footer", this has been implemented already and we change anything there we also have to rework on that implementation and i would highly avoid that.
Tell me if you feel confident about these remarks, @gingapa, if you feel that is not clear enough we can talb about it, otherwise i can take this over and complete it, we need to do this this week, including the implementation.